### PR TITLE
 Updated vu-ams doc with OS3 compatible plugin (try 2)

### PR DESCRIPTION
**Pull request by JdenHartog**
try 2 of : #66 
:blush:

### CODE REVIEW

**Comment by smathot** [comment]
Perfect! I made [some suggestions](https://github.com/JdenHartog/vu_ams/issues/1) on how to improve the plugin, one of which affects the API. So--if you agree--I'll wait before merging this pull request.

File: content/pages/manual/devices/vuams.md [hunk 1]
```diff
@@ -1,8 +1,102 @@
...
+ source: VU-AMS.png
+ caption: |
+  The VU-AMS device.
+--%
+
+[TOC]
+
+## Using the `vu-ams` plugin
+
+Parallel_port_trigger is a third-party plugin.
```
Comment (smathot) [inline content/pages/manual/devices/vuams.md]: Let's make that VU_ams

Comment (JdenHartog) [inline content/pages/manual/devices/vuams.md]: Oops.... 
It clearly shows I started out with the Parallel_port_trigger plugin as a template :innocent:

**Comment by smathot** [comment]
Closing due to inactivity

**Pull request closed without merging**